pytorch_lightning/core/lightning.py

```diff
@@ -1464,19 +1466,34 @@ def __init__(self, hparams):
                 If your checkpoint saved a GPU model and you now load on CPUs
                 or a different number of GPUs, use this to map to the new setup.
                 The behaviour is the same as in :func:`torch.load`.
-            tags_csv: Optional path to a .csv file with two columns (key, value)
+            hparams_file: Optional path to a .yaml file with hierarchical structure.
                 as in this example::
 
-                    key,value
-                    drop_prob,0.2
-                    batch_size,32
+                    model:
+                        drop_prob: 0.2
+                    dataloader:
+                        batch_size: 32
 
                 You most likely won't need this since Lightning will always save the hyperparameters
                 to the checkpoint.
                 However, if your checkpoint weights don't have the hyperparameters saved,
-                use this method to pass in a .csv file with the hparams you'd like to use.
-                These will be converted into a :class:`~argparse.Namespace` and passed into your
+                use this method to pass in a .yaml file with the hparams you'd like to use.
+                These will be converted into a :class:`~dict` and passed into your
                 :class:`LightningModule` for use.
```

Comment:
maybe make an emphasizing note here that due to yml structure only hparams in dict structure are supported. If the model accepts a argparse Namespace, it would require the user to refactor their code.

Reply:
I will add a description like:
```
If your model's `hparams` argument is :class:`~argparse.Namespace`
and .yaml file has multi-level structure, you need to refactor your model to treat
`hparams` as :class:`~dict`.
```

Is it okay?

Reply:
saw your doc updates, nice, thanks!